docs/source/dev/backends.rst

```diff
@@ -363,7 +363,7 @@ The following tables list the functions available in the `CUDA Runtime API <http
     +----------------------------+--------------------------------------------------------------------------------------------+
     | cudaGetSymbolSize          | --                                                                                         |
     +----------------------------+--------------------------------------------------------------------------------------------+
-    | cudaHostAlloc              | alpaka::allocMappedBuf<TPlatform, TElement>(host, extents) 1D, 2D, 3D supported!           |
+    | cudaHostAlloc              | alpaka::allocMappedBuf<TElement, TIdx>(host, extents) 1D, 2D, 3D supported!                |
```

Comment:
Sry my comment was wrong this should be `allocMappedBufIfSupported`. There is no strong need that this memory is mapped.
`allocMappedBuf` is not available for any accelerator.